portability-spi-api/src/main/java/org/datatransferproject/spi/api/types/AuthFlowConfiguration.java

```diff
@@ -6,34 +6,46 @@
 import org.datatransferproject.types.transfer.PortableType;
 import org.datatransferproject.types.transfer.auth.AuthData;
 
+import static org.datatransferproject.types.common.PortabilityCommon.AuthProtocol;
+
 /**
  * Configuration for an authorization flow. A flow has an initial URL and optional initial
  * authentication data.
  */
 @JsonTypeName("org.dataportability:AuthFlowConfiguration")
 public class AuthFlowConfiguration extends PortableType {
   private final String url;
+  private final String tokenUrl;
+  private final AuthProtocol authProtocol;
   private AuthData initialAuthData;
 
   /**
    * Ctor used when the flow does not require initial authentication data.
    *
    * @param url the initial URL.
    */
-  public AuthFlowConfiguration(String url) {
+  public AuthFlowConfiguration(String url, String tokenUrl, AuthProtocol authProtocol) {
```

Comment:
Let's put tokenUrl at the end to not have two strings in a row

Reply:
Good idea, done